### PR TITLE
PLANET-7055 Setup Playwright

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-7055](https://jira.greenpeace.org/browse/PLANET-7055)

**Related PR:** https://github.com/greenpeace/planet4-master-theme/pull/1908

### Testing

On local, run `npm install` to install Playwright and its needed dependencies, and then to run the tests you can use the following commands:

- `npx playwright test` for all tests
- `npx playwright test tests/e2e/404.spec.js` for individual tests (added this very simple one for now just to test the setup)

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [iocaste](https://www-dev.greenpeace.org/test-iocaste) | [admin](https://www-dev.greenpeace.org/test-iocaste/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-iocaste/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-iocaste) | [composer-local.json](https://github.com/greenpeace/planet4-test-iocaste/blob/main/composer-local.json)

:watch: 2023.02.02 14:53:40

File: classes/class-loader.php [hunk 1]
```diff
@@ -454,6 +454,9 @@ public function enqueue_public_assets() {
 			'enable_analytical_cookies'  => $option_values['enable_analytical_cookies'] ?? '',
 			'enable_google_consent_mode' => $option_values['enable_google_consent_mode'] ?? '',
 			'cookies_default_copy'       => self::get_cookies_default_copy(),
+			'cookies_field'              => $option_values['cookies_field'],
```
[the remark this answers is not in the document]
Comment (lithrel) [reply classes/class-loader.php:457]: I see an alternative in either going in the admin settings to read it first, or creating an authenticated rest endpoint listing those options, but that's something we can do later, when we have more need for matching settings to frontend output :ok_hand:

**Review by lithrel — APPROVED** [review]
:+1:

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**